verification/curator-service/ui/src/components/NewCaseForm.test.tsx

```diff
@@ -27,38 +19,3 @@ it('renders form', () => {
     expect(getByText(/Source URL/i)).toBeInTheDocument();
     expect(getByText(/Nationality/i)).toBeInTheDocument();
 });
-
-it('submits case ok', async () => {
-    const axiosResponse = {
-        data: {},
-        status: 200,
-        statusText: 'OK',
-        config: {},
-        headers: {},
-    };
-    mockedAxios.post.mockResolvedValueOnce(axiosResponse);
-
-    const { getByText } = render(<NewCaseForm user={user} />);
-    expect(getByText(/Submit case/i)).toBeInTheDocument();
-
-    await wait(() => {
-        fireEvent.click(getByText(/Submit case/));
-    });
-    expect(mockedAxios.post).toHaveBeenCalledTimes(1);
-});
-
-it('submits case not ok', async () => {
```

Comment:
we should be testing for a POST vs PUT really in those tests depending on if we had an initial case or not, I thought I added that but no?

Reply:
Those checks aren't in this test file in the master branch.

We do check in cypress tests if it's a POST or PUT by which route we wait on, comparing NewCaseFormTest and EditCaseTest

Reply:
ok nm then, thanks